### PR TITLE
removes params and creds from logger

### PR DESCRIPTION
**Pull request by Wmiranda1**
Removes logging parameters and credentials cred for posttool

### CODE REVIEW

File: src/main/java/com/aquent/viewtools/PostTool.java [hunk 1]
```diff
@@ -88,9 +88,9 @@ public Credentials createCreds(String user, String pass) {
    * @param creds - A credentials object for authenticated requests.
    * @return A PostToolResponse Object
    */
-  public PostToolResponse sendStringData(String url, String data, String method, String contentType, Credentials creds) {
+  public PostToolResponse sendStringData(String url, String data, String method, String contentType) {
```
Comment (cfalzone) [inline src/main/java/com/aquent/viewtools/PostTool.java]: we don't want to remove the creds/params from the method signatures.  only the logging statements

File: src/main/java/com/aquent/viewtools/PostTool.java [hunk 2]
```diff
@@ -229,9 +229,9 @@ public PostToolResponse sendStringData(String url, String data) {
    * @param creds - A credentials object for authenticated requests.
    * @return A PostToolResponse Object
    */
-  public PostToolResponse send(String url, Map<String, String> params, String method, String contentType, Credentials creds) {
-    Logger.debug(this, "send(Map) called with url=" + url + ", params=" + params + ", and method=" + method
-        + ", contentType = " + contentType + ", creds=" + UtilMethods.isSet(creds));
+  public PostToolResponse send(String url, Map<String, String method, String contentType) {
```
Comment (cfalzone) [inline src/main/java/com/aquent/viewtools/PostTool.java]: Not quite right here .. this is where having an IDE would help.  Again, not removing the parameters from the method, just the logging statements, but this line straight up won't compile.

File: src/main/java/com/aquent/viewtools/PostTool.java [hunk 3]
```diff
@@ -322,9 +322,9 @@ public PostToolResponse send(String url, Map<String, String> params, String meth
    * @return A PostToolResponse Object
    * @throws Exception from URLDecoder.decode
    */
-  public PostToolResponse send(String url, String params, String method, String contentType, Credentials creds) throws Exception {
-    Logger.debug(this, "send(String) called with url=" + url + ", params=" + params + ", and method=" + method
-        + ", contentType = " + contentType + ", creds=" + UtilMethods.isSet(creds));
+  public PostToolResponse send(String url, String method, String contentType) throws Exception {
```
Comment (cfalzone) [inline src/main/java/com/aquent/viewtools/PostTool.java]: Also here, don't remove the parameters from the method signature.

**Review by cfalzone — CHANGES_REQUESTED** [review]
See Comments

**Review by cfalzone — APPROVED** [review]
LG2M -- don't forget -- Don't merge in github use git flow on the CLI ;)

File: src/main/java/com/aquent/viewtools/PostTool.java [hunk 4]
```diff
@@ -90,7 +90,7 @@ public Credentials createCreds(String user, String pass) {
    */
   public PostToolResponse sendStringData(String url, String data, String method, String contentType, Credentials creds) {
     Logger.debug(this, "sendStringData called with url=" + url + ", data=" + data + ", and method=" + method
```
Comment (rediris) [inline src/main/java/com/aquent/viewtools/PostTool.java:92]: Is the goal to remove the entire `Logger.debug()` statement?

File: src/main/java/com/aquent/viewtools/PostTool.java [hunk 5]
```diff
@@ -230,8 +230,8 @@ public PostToolResponse sendStringData(String url, String data) {
    * @return A PostToolResponse Object
    */
   public PostToolResponse send(String url, Map<String, String> params, String method, String contentType, Credentials creds) {
-    Logger.debug(this, "send(Map) called with url=" + url + ", params=" + params + ", and method=" + method
-        + ", contentType = " + contentType + ", creds=" + UtilMethods.isSet(creds));
+    Logger.debug(this, "send(Map) called with url=" + url + ", and method=" + method
```
Comment (rediris) [inline src/main/java/com/aquent/viewtools/PostTool.java:233]: @cfalzone , is the goal to remove the entire `Logger.debug()` statement?

Comment (cfalzone) [reply src/main/java/com/aquent/viewtools/PostTool.java:233]: Nope just the parameters, that is where the username and passwords were showing up in logs

**Pull request merged**